README.md

```diff
@@ -58,6 +58,9 @@ REST_FRAMEWORK = {
     "PAGE_SIZE": 100,
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
     "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAuthenticated",
+    ],
```

Comment:
This line is a duplicate of the above 😀

Reply:
but yours is better formatted so let's keep it

Reply:
Whoops! Not sure what happened here. Thanks! 721224db68f29b3edb252a0b9d6e3171d5fb9adb